pkgs/tools/text/ripgrep/default.nix

```diff
@@ -1,23 +1,28 @@
-{ stdenv, fetchFromGitHub, rustPlatform, makeWrapper }:
+{ stdenv, fetchFromGitHub, rustPlatform, makeWrapper, asciidoc, libxslt }:
 
-with rustPlatform;
-
-buildRustPackage rec {
+rustPlatform.buildRustPackage rec {
   name = "ripgrep-${version}";
-  version = "0.7.1";
+  version = "0.8.0";
 
   src = fetchFromGitHub {
     owner = "BurntSushi";
     repo = "ripgrep";
     rev = "${version}";
-    sha256 = "0z3f83vhy464k93bc55i9lr6z41163q96if938p9ndhx2q3a20ql";
+    sha256 = "0ha46vhma2diqxfgasdc9gwlkcrlhc4l65nayvdy4m2ah5bm4qp6";
   };
 
-  cargoSha256 = "1d6s01gmyfzb0vdf7flq6nvlapwcgbj0mzcprzyg4nj5gjkvznrn";
+  cargoSha256 = "0q44qa9myrzg42q0lvclpk5nypmf0q7v3xy5nnsb28j3imvcrs4p";
+
+  nativeBuildInputs = [ asciidoc libxslt ];
 
   preFixup = ''
     mkdir -p "$out/man/man1"
-    cp "$src/doc/rg.1" "$out/man/man1"
+    cp target/release/build/ripgrep-*/out/rg.1 "$out/man/man1/"
+
+    mkdir -p "$out/share/"{bash-completion/completions,fish/completions,zsh/site-functions}
+    cp target/release/build/ripgrep-*/out/rg.bash "$out/share/bash-completion/completions/"
+    cp target/release/build/ripgrep-*/out/rg.fish "$out/share/fish/completions/"
```

Comment:
Can you install the fish completions on `/share/fish/vendor_completions.d/` ? It's where they are expected: https://github.com/NixOS/nixpkgs/blob/57707ea2caf1b1898c7f8bec59372052ed7e2146/nixos/modules/programs/fish.nix#L168-L172

Reply:
Thanks!